src/mumble/MainWindow.h

```diff
@@ -103,6 +103,10 @@ class MainWindow : public QMainWindow, public MessageHandler, public Ui::MainWin
 
 		QPoint qpContextPosition;
 
+		MUComboBox *qcbTransmitMode;
+		QAction *qaTransmitMode;
+		QAction *qaTransmitModeSeparator;
+
```

Comment:
What's the reason for moving these fields?

Reply:
I couldn't access `qcbTransitMode` from DBus because it was protected. Because I was still playing around I just made all three public, though the actions can probably go back to protected.

Reply:
Ah okay. I think I'd prefer to leave these fields protected and add public getter and setter functions instead. That way this implementation is not dependent on what is used internally :point_up:

Reply:
I think I just looked at how muting/deafening worked and copied that, without looking further in what the combobox does. It does look much cleaner now.